app/facades/food_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodFacade
+  class << self
+    def food_search(food)
+      data = FoodService.find_foods(food)
+      data[:foods].map do |food_data|
+        Food.new(food_data)
+      end.take(10)
+    end
+
+    def result_count(food)
+      data = FoodService.find_foods(food)
+      data[:totalHits]
+    end
```

Comment:
- this is creating a lot of Food PORO instances that will never be used/returned to the controller.
- this is hitting the same API endpoint twice. Ideally, we would get both data points with a single call.
```
def food_search(food)
  data = FoodService.find_foods(food)
  total_hits = data[:totalHits]
  foods = data[:foods].first(10).map do |food_data|
    Food.new(food_data)
  end
  [total_hits, foods]
end
```